airflow/providers/apache/drill/example_dags/example_drill_dag.py

```diff
@@ -19,14 +19,15 @@
 """
 Example Airflow DAG to execute SQL in an Apache Drill environment using the `DrillOperator`.
 """
+from datetime import datetime
+
 from airflow.models import DAG
 from airflow.providers.apache.drill.operators.drill import DrillOperator
-from airflow.utils.dates import days_ago
 
 with DAG(
     dag_id='example_drill_dag',
     schedule_interval=None,
-    start_date=days_ago(2),
+    start_date=datetime(2021, 1, 1),
```

Comment:
Should we use `START_DATE` here like https://github.com/apache/airflow/pull/18657/files#diff-80e53751469787f4571bdcf652c0803676ab5e19b91c1dab7a39c678e9e7bfe3R76, so it will be consistent for all example DAGs?

Reply:
I chose to create a `START_DATE` variable in that Apache Beam example DAG file because there are several DAGs declared in that single file -- all of which had the same `start_date` value.  IMO it doesn't add much value to do the same here as there is only one DAG declared so `start_date` is set once.